diff --git a/node_modules/datatables.net/js/jquery.dataTables.mjs b/node_modules/datatables.net/js/jquery.dataTables.mjs
index 04bee9d..c83bb55 100644
--- a/node_modules/datatables.net/js/jquery.dataTables.mjs
+++ b/node_modules/datatables.net/js/jquery.dataTables.mjs
@@ -1636,9 +1636,10 @@ DataTable.util = {
 	 *  @returns {string} escaped string
 	 */
 	escapeRegex: function ( val ) {
-		return val.replace( _re_escape_regex, '\\$1' );
+		return val.replace( /[-\/\\^$*+?.()|[\]{}]/g, '\\$&' );
 	},
 
+
 	/**
 	 * Create a function that will write to a nested object or array
 	 * @param {*} source JSON notation string
