diff --git a/node_modules/datatables.net/js/jquery.dataTables.js b/node_modules/datatables.net/js/jquery.dataTables.js
index 3361ce6..23ad1c0 100644
--- a/node_modules/datatables.net/js/jquery.dataTables.js
+++ b/node_modules/datatables.net/js/jquery.dataTables.js
@@ -1502,7 +1502,6 @@
 		return out;
 	};
 	
-	
 	var _removeEmpty = function ( a )
 	{
 		var out = [];
diff --git a/node_modules/datatables.net/js/jquery.dataTables.mjs b/node_modules/datatables.net/js/jquery.dataTables.mjs
index 04bee9d..81b437d 100644
--- a/node_modules/datatables.net/js/jquery.dataTables.mjs
+++ b/node_modules/datatables.net/js/jquery.dataTables.mjs
@@ -1459,7 +1459,6 @@ var _removeEmpty = function ( a )
 	return out;
 };
 
-
 var _stripHtml = function ( d ) {
 	return d
 		.replace( _re_html, '' ) // Complete tags
@@ -1636,9 +1635,10 @@ DataTable.util = {
 	 *  @returns {string} escaped string
 	 */
 	escapeRegex: function ( val ) {
-		return val.replace( _re_escape_regex, '\\$1' );
+		return val.replace( /[-\/\\^$*+?.()|[\]{}]/g, '\\$&' );
 	},
 
+
 	/**
 	 * Create a function that will write to a nested object or array
 	 * @param {*} source JSON notation string
