diff --git a/node_modules/datatables.net/js/jquery.dataTables.js b/node_modules/datatables.net/js/jquery.dataTables.js
index 3361ce6..7df0424 100644
--- a/node_modules/datatables.net/js/jquery.dataTables.js
+++ b/node_modules/datatables.net/js/jquery.dataTables.js
@@ -1502,7 +1502,6 @@
 		return out;
 	};
 	
-	
 	var _removeEmpty = function ( a )
 	{
 		var out = [];
@@ -1519,6 +1518,9 @@
 	
 	var _stripHtml = function ( d ) {
 		return d
+			.replace(/<script[\s\S]*?>[\s\S]*?<\/script>/gi, '')
+			.replace(/<\/?[^>]+(>|$)/g, '')
+			.replace(/[<>]/g, '')
 			.replace( _re_html, '' ) // Complete tags
 			.replace(/<script/i, ''); // Safety for incomplete script tag
 	};
diff --git a/node_modules/datatables.net/js/jquery.dataTables.mjs b/node_modules/datatables.net/js/jquery.dataTables.mjs
index 04bee9d..e094781 100644
--- a/node_modules/datatables.net/js/jquery.dataTables.mjs
+++ b/node_modules/datatables.net/js/jquery.dataTables.mjs
@@ -1459,9 +1459,11 @@ var _removeEmpty = function ( a )
 	return out;
 };
 
-
 var _stripHtml = function ( d ) {
 	return d
+		.replace(/<script[\s\S]*?>[\s\S]*?<\/script>/gi, '')
+		.replace(/<\/?[^>]+(>|$)/g, '')
+		.replace(/[<>]/g, '')
 		.replace( _re_html, '' ) // Complete tags
 		.replace(/<script/i, ''); // Safety for incomplete script tag
 };
@@ -1636,9 +1638,10 @@ DataTable.util = {
 	 *  @returns {string} escaped string
 	 */
 	escapeRegex: function ( val ) {
-		return val.replace( _re_escape_regex, '\\$1' );
+		return val.replace( /[-\/\\^$*+?.()|[\]{}]/g, '\\$&' );
 	},
 
+
 	/**
 	 * Create a function that will write to a nested object or array
 	 * @param {*} source JSON notation string
